Lib/fontTools/colorLib/geometry.py

```diff
@@ -1,19 +1,15 @@
 """Helpers for manipulating 2D points and vectors in COLR table."""
 
-from math import copysign, cos, sqrt, pi
+from math import copysign, cos, hypot, pi
 from fontTools.misc.fixedTools import otRound
 
 
 def _vector_between_points(a, b):
     return (b[0] - a[0], b[1] - a[1])
 
 
-def _vector_length(vec):
-    return sqrt(vec[0] * vec[0] + vec[1] * vec[1])
-
-
 def _distance_between_points(a, b):
-    return _vector_length(_vector_between_points(a, b))
+    return hypot(*_vector_between_points(a, b))
```

Comment:
given hypot I wonder if we even need this function?

Reply:
i removed it